plugins/Name_Multiple.py

```diff
@@ -35,16 +35,16 @@ def init(self, logger):
             title = T_('The name tag contains two names'),
             detail = T_(
 '''The tag `name=*` contains multiple names, separated by a semicolon,
-a "/", a "\\" or a "+". This issue was probably produced by the fusion of two
-way and the concatenation of the names of the streets.'''),
+a "/", a "\\" or a "+". This issue was probably produced by merging two
+ways and the concatenation of the names of the streets.'''),
```

Comment:
This sentence looks a bit strange.

Should it be rewritten as following?

by merging two ways and concatenating the names of streets

Reply:
I think the combination of the names happens automaticly when you merge two ways, so in this case it's a noun, not a verb (as at the start of this sentence). But, I have the same concern. I would simply remove the second part of the sentence altogether ("and the concatenation of the names of the streets").

Reply:
I see, and you're right.

We can remove the second part, or rewrite with "merging two ways, which would concatenate the name of the streets"